src/common/interfaces.ts

```diff
@@ -159,6 +159,11 @@ export enum Grid {
   TWO_ON_ONE = '2x1',
 }
 
+export interface IJobResumeState {
```

Comment:
i think 'type' will do the work here, as it reference more to object build and no extent or implement of an class